### PR TITLE
Soap holoprojector for clown

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

add holoprojector for clown, that create soap projections

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added new holoprojector for clown!

### CODE REVIEW

**Comment by ficcialfaint** [comment]
The copyright field in `meta.json`s should be either a commit link or a permalink.
Also something weird with your tabulation in meta's

File: Resources/Prototypes/Entities/Structures/Holographic/projections.yml
Comment (Kit0vras) [inline Resources/Prototypes/Entities/Structures/Holographic/projections.yml:1]: it's just a projection without any functionality, so it doesn't make sense.
add slippery and fixtures (to make slippery work)

**Comment by Kit0vras** [comment]
> it will also be good if you add at least a small screenshot or video for clarity of the added

**Comment by lzk228** [comment]
> it will also be good if you add at least a small screenshot or video for clarity of the added

Unfortunately, i cant

**Comment by brainfood1183** [comment]
Looks good, how will the clown get their hands on the holosoap? Is it to be added to theatre? random loot in maints or research based?

**Comment by lzk228** [comment]
> Looks good, how will the clown get their hands on the holosoap? Is it to be added to theatre? random loot in maints or research based?

i changed clown startgear

**Comment by EmoGarbage404** [comment]
honestly the clown gear slide is real. They now start with a horn, laughter machine, and an infinite soap machine. Some of it should be shoved in a vendor, crate, or sci unlock.

**Comment by brainfood1183** [comment]
> honestly the clown gear slide is real. They now start with a horn, laughter machine, and an infinite soap machine. Some of it should be shoved in a vendor, crate, or sci unlock.

I have a toy box crate pr that pjb approved for merging. It adds a toybox and some new toys. I guess the recorder could be moved to it.

**Comment by deltanedas** [comment]
would actually be very powerful for cleaning blood of clothes wsci

**Review by AJCM-git — CHANGES_REQUESTED** [review]
Move this to a tech or something, also this needs a new "RemovePreviousProjection" bool datafield to HolosignProjectorComponent that removes the previous projection that is false by default, then set it to true for this for.

**Comment by lzk228** [comment]
omg

**Pull request closed without merging**